centreon/src/Core/Host/Application/UseCase/AddHost/AddHostValidation.php

```diff
@@ -73,7 +75,14 @@ public function __construct(
      */
     public function assertIsValidName(string $name): void
     {
+        Assertion::regex(
+            $name,
+            '/^[^' . MonitoringServer::ILLEGAL_CHARACTERS . ']*$/',
+            'Host::name'
+        );
```

Comment:
```suggestion
        Assertion::unauthorizedCharacters(
            $name,
            MonitoringServer::ILLEGAL_CHARACTERS,
            'Host::name'
        );
```
This assert will have a clearer (and consistant with other endpoints) error message.
(sorry, I didn't check which assertion was used in other files when I made my first request change 🥲 )

Reply:
Thank you! it's done